utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -0,0 +1,93 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.request.target.CustomTarget
+import com.bumptech.glide.request.transition.Transition
+import javax.inject.Inject
+
+// TODO(#169,#277): Replace this with exploration asset downloader and add test cases for loading image
```

Comment:
Let's frame these as 2 TODOs rather than combining them since they are separate tasks to complete.

Reply:
okay. done.